File: generateclasses/Prixcategoriebrute_v.java
package entite;

public class Prixcategoriebrute_v{
    int idcategorie; 
    int idstyle; 
    int idtaille; 
    double prixtotal; 

    public Prixcategoriebrute_v(){ }
    
    public int getIdcategorie(){
        return this.idcategorie;
    }
    public void setIdcategorie(int idcategorie){
        this.idcategorie=idcategorie;
    }
    public int getIdstyle(){
        return this.idstyle;
    }
    public void setIdstyle(int idstyle){
        this.idstyle=idstyle;
    }
    public int getIdtaille(){
        return this.idtaille;
    }
    public void setIdtaille(int idtaille){
        this.idtaille=idtaille;
    }
    public double getPrixtotal(){
        return this.prixtotal;
    }
    public void setPrixtotal(double prixtotal){
        this.prixtotal=prixtotal;
    }

}

File: generateclasses/Prixcategoriebrute_v.cs
namespace entite;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("prixcategoriebrute_v")]
public class Prixcategoriebrute_v : Motherobj<Prixcategoriebrute_v>  {
    [Column("idcategorie")]
    public int idcategorie {get; set;} 
    [Column("idstyle")]
    public int idstyle {get; set;} 
    [Column("idtaille")]
    public int idtaille {get; set;} 
    [Column("prixtotal")]
    public double prixtotal {get; set;} 

    public Prixcategoriebrute_v(){ }
    

}
